index.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+const child_process = require("child_process");
+const minimist = require("minimist");
+
+const {runChecks, getCommands} = require("./src/utils");
+
+var argv = require('minimist')(process.argv.slice(2));
+
+try {
+    runChecks();
+    const commands = getCommands();
+    if (commands.length === 0) {
+        console.log("all packages up to date");
+        process.exit(0);
+    }
+
+    for (const command of commands) {
+        if (!argv["dry-run"]) {
+            console.log(`running: ${command}`);
+            child_process.execSync(command, {encoding: "utf8"});
+        } else {
+            console.log(`would run: ${command}`);
+        }
+    }
+
+    if (!argv["dry-run"]) {
+        console.log("pushing tags");
+        child_process.execSync("git push --tags");
+    } else {
+        console.log(`would run: git push --tags`);
+    }
```

Comment:
Feel like the text `git push --tags` should be in a constant, and then there should be an `maybeRunCommand` command that takes that string like this:

```
function (command, description) {
    console.log(description);
    const isDryRun = argv["dry-run"];
    if (isDryRun) {
        console.log(`would run: ${command}`);
    } else {
        child_process.execSync(command);
    }
}